.deployment/barista/Jenkinsfile

```diff
@@ -0,0 +1,212 @@
+pipeline {
+  agent none
+
+  options {
+    ansiColor("xterm")
+    timeout(time: 4, unit: "HOURS")
+    timestamps()
+  }
+
+  parameters {
+    booleanParam(
+      name: "INTERNAL_VERSION",
+      defaultValue: true,
+      description: "Build the internal version of Barista"
+    )
+  }
+
+  environment {
+    WORKSPACE_DIR = "/home/dynatrace"
+    BARISTA_PATH = "barista"
+    ICONS_PATH = "barista-icons"
+    FRONTEND_IMAGE_NAME = "designops/barista-app-frontend"
+    BITBUCKET_URL = credentials("bitbucket-url")
+    DOCKER_REGISTRY = credentials("docker-registry")
+  }
+
+  stages {
+
+    stage("Build Barista") {
+      agent {
+        node {
+          label "k8s-barista-builder"
+        }
+      }
+
+      stages {
+        stage("Checkout barista icons") {
+          steps {
+            checkout changelog: false, poll: false, scm: [
+              $class: "GitSCM",
+              branches: [[name: "*/master"]],
+              doGenerateSubmoduleConfigurations: false,
+              extensions: [[$class: "RelativeTargetDirectory", relativeTargetDir: "$ICONS_PATH"]],
+              submoduleCfg: [], userRemoteConfigs: [[url: "${BITBUCKET_URL}scm/wx/barista-icons.git" ]]
+            ]
+          }
+        }
+
+        stage("Add internal Icons") {
+          when {
+            expression { return params.INTERNAL_VERSION }
+          }
+          steps {
+            dir(WORKSPACE_DIR) {
+              sh '''
+                rm -rf ./node_modules/@dynatrace/barista-icons
+                cp -R ./dist/tmp/barista-icons ./node_modules/@dynatrace/barista-icons
+              '''
+            }
+          }
+        }
+
+        stage("Prepare Workspace") {
+          steps {
+            sh'''
+              files="node_modules package-lock.sha1"
+
+              for file in $files
+              do
+                ln -s "$WORKSPACE_DIR/$file" "$PWD/$BARISTA_PATH/$file"
+              done
+
+            '''
+
+            dir(BARISTA_PATH) {
+              configFileProvider([
+                configFile(fileId: "barista-env", variable: "ENV")
+              ]) {
+                sh '''
+                  mv "$ENV" .env
+
+                  # Set public build to the inverse of INTERNAL_VERSION
+                  if $INTERNAL_VERSION ; then
+                    PUBLIC_BUILD=false
+                  else
+                    PUBLIC_BUILD=truedocker
```

Comment:
is this supposed to be `truedocker` ? I guess not? :D

Reply:
haha nope :D typo :D